src/rust/generators/sysmon-generator/src/serialization.rs

```diff
@@ -1,6 +1,7 @@
 use std::str::FromStr;
 
 use grapl_service::decoder::decompress::PayloadDecompressionError;
+use prost;
```

Comment:
Can also be removed.

Reply:
Ah, nvm just saw your comment above. Will fix.